website/docs/components/table/partials/code/how-to-use.md

```diff
@@ -25,7 +27,7 @@ If you have your own content and don't want to use a model, you can still benefi
 </Hds::Table>
```

Comment:
Question: for some of these examples do we want to use the code block to show the rendered component? Might be tough in this layout, but just curious

Reply:
we're talking about it 👍